Source/SteveSloth/Private/MyPlayer.cpp

```diff
@@ -143,6 +148,26 @@ void AMyPlayer::AddGrapplingHook()
 	bIsGrapplingHookUnlocked = true;
 }
 
+void AMyPlayer::AddClimbingClaw()
+{
+	bIsClimbingClawUnlocked = true;
+}
+
+void AMyPlayer::AddShovel()
+{
+	bIsShovelUnlocked = true;
+}
+
+void AMyPlayer::AddPropeller()
+{
+	bIsPropellerUnlocked = true;
+}
+
+void AMyPlayer::AddMagnetic()
+{
+	bIsMagneticUnlocked = true;
+}
+
```

Comment:
All of these from 146 to 170 should just be a list of upgradeable items that one function will check if the item is unlocked or not